search/django/requirements.txt

```diff
@@ -6,4 +6,4 @@ celery==4.4.7
 django-celery-results
 django-celery-beat
 redis
-https://github.com/Sydney-Informatics-Hub/Weed-ID-Interchange/archive/ingestion.zip
\ No newline at end of file
+https://github.com/Sydney-Informatics-Hub/Weed-ID-Interchange/archive/image-link-dataset.zip
```

Comment:
This will need to become `master` upon merge, no?

Reply:
after merging yes